docs/source/blog/6-costly-data-labeling-mistakes-and-how-to-avoid-them.md

```diff
@@ -0,0 +1,127 @@
+
+Let’s take a look at an image pulled from MIT’s [Label Errors](https://labelerrors.com/) website. The image was labeled by the ImageNet algorithm, so we don’t actually know which annotation error(s) led to the algorithm’s incorrect prediction, but it can give us a glimpse of why labels get missed.
+<br/>  
+
+<img src="/images/6-mistakes/great-dane-label-error.png" alt="Image of label studio timeline" style="max-width: 20rem;" />
+
+*Image Source: [Label Errors](https://labelerrors.com/)*
+
+At first glance, it’s hard to tell how many dogs are in the picture. The dalmatian in the foreground is clear, but the Great Dane’s face is obscured. What’s more, there are two *additional* dogs in the photo; they’re just harder to spot. If you gave this image to an annotator and told them to place a bounding box around every dog, it’s reasonable to assume your annotator might miss one or two dogs if they were fatigued or moving quickly. 
+
+### The Fix: Use the Consensus Method
+
+The key to catching and mitigating this kind of error is a strong, collaborative data validation process. The consensus method works great: Ask two or more different annotators to label the same sample of images, find the classes where annotators disagree, and revise your instructions until the annotators are able to find consistency. A thorough peer review process can also catch many quality errors. 
+
+## 2. Incorrect Fit
+
+Sometimes the bounding box simply isn’t tight enough around the object. The gaps that are left can add noise to the data, harming the model’s ability to identify the precise object. If every bounding box for a bird contains both the bird and large chunks of blue sky, the algorithm might start to assume the blue sky is a necessary part of being a bird. It would be less likely, then, to correctly identify the object in this image: 
+<br/>
+
+<img src="/images/6-mistakes/good-bad-label-error.png" alt="Image of label studio timeline" class="gif-border" />
+
+*Image Source: [ResearchGate](https://www.researchgate.net/figure/Good-left-and-bad-right-bounding-boxes-for-the-object-fox-judged-on-both-coverage-as_fig15_301844872)*
+
+Another kind of incorrect fit is the occlusion error. Expectations are a powerful thing, and our brains [love to fill in gaps for us](https://www.interaction-design.org/literature/topics/gestalt-principles). This tendency to “complete the picture” manifests in the occlusion error: when an annotator puts a box around the expected size of the complete object instead of sticking to the visible part.
+
+### The Fix: Provide Clear Instructions
+
+You’ll likely find that unclear instructions are at the root of this error. In the instructions you provide for your labelers, clearly define the tolerance/accuracy you’re looking for. Use supporting screengrabs or videos to illustrate what you mean (“good” and “bad” examples are an especially handy way to get the point across). 
```

Comment:
Replace line #48 with the following: 

You’ll likely find that unclear instructions are at the root of this error. In the instructions for your labelers, clearly define the tolerance/accuracy you’re looking for. Use supporting screen recordings or videos to illustrate what you mean (“good” and “bad” examples are an especially handy way to get the point across). 
 
Add a suggestion, <Cmd+g>
Add heading textAdd bold text, <Cmd+b>Add italic text, <Cmd+i>
Add a quote, <Cmd+Shift+.>Add code, <Cmd+e>Add a link, <Cmd+k>
Add a bulleted list, <Cmd+Shift+8>Add a numbered list, <Cmd+Shift+7>Add a task list, <Cmd+Shift+l>
Directly mention a user or team
Reference an issue, pull request, or discussion